src/test/CMakeLists.txt

```diff
@@ -45,19 +45,23 @@ endmacro()
 ## example: add_shadow_tests(BASENAME bind METHODS ptrace preload LOGLEVEL debug ARGS --pin-cpus)
 ## will create two tests named bind-shadow-ptrace and bind-shadow-preload
 macro(add_shadow_tests)
-   cmake_parse_arguments(SHADOW_TEST "" "BASENAME;LOGLEVEL" "METHODS;ARGS;CONFIGURATIONS;PROPERTIES" ${ARGN})
+    cmake_parse_arguments(SHADOW_TEST "" "BASENAME;LOGLEVEL" "SHADOW_CONFIG;METHODS;ARGS;CONFIGURATIONS;PROPERTIES" ${ARGN})
```

Comment:
I think `SHADOW_CONFIG` takes a single argument, so it should be grouped with the `one_value_keywords` list (`BASENAME;LOGLEVEL`) rather than the `multi_value_keywords` list (`METHODS;ARGS;CONFIGURATIONS;PROPERTIES`). See [the docs for cmake_parse_arguments](https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html).

Reply:
Done